src/source/src/manager.rs

```diff
@@ -363,7 +365,10 @@ pub mod test_utils {
             properties: Default::default(),
             info,
             source_manager,
-            connector_node_addr: "127.0.0.1:60061".to_string(),
+            connector_params: ConnectorParams {
+                connector_source_endpoint: Some("127.0.0.1:60061".to_string()),
```

Comment:
We can also set None for the source endpoint, so that we can test the scenario where the source endpoint is not set. If we fail any unit test when we set the source endpoint to None, we should fix it.